release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -72,6 +72,27 @@ For more information, see xref:../installing/installing_aws/installing-aws-vpc.a
 [id="ocp-4-14-web-console"]
 === Web console
 
+[id="ocp-4-14-administrator-perspective"]
+==== Administrator Perspective
+
+With this release, there are several updates to the *Administrator* perspective of the web console. You can:
+
+* Narrow down the list of resources in a list view or search page with exact search capabilities. This will help when you have similarly named resources and fuzzy search doesn't narrow down your search.
+* Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and click *Share Feedback* from the drop-down list.
```

Comment:
Suggestion for both instances of "click" to be in the present progressive tense:

```suggestion
* Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and clicking *Share Feedback* from the drop-down list.
```